### PR TITLE
chore: Fix several supported version things

### PR DESCRIPTION
**Pull request by chapterjason**

### CODE REVIEW

File: src/config/parameters.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-// If sf 3.3 and no annotations is installed, then disable it.
+// If symfony is greater than 3.3 and no annotations are installed, then disable it.
 if (\Symfony\Component\HttpKernel\Kernel::VERSION_ID >= 30300 && !class_exists('Doctrine\Common\Annotations\Annotation')) {
```
Comment (Nyholm) [inline src/config/parameters.php]: Do we still need this if statement? Are there others like it?

**Review by Nyholm — APPROVED** [review]
Thank you. 

JUst make sure to update phpstan baseline so it is happy

**Comment by chapterjason** [comment]
@Nyholm Always miss these cause I try to test lowest locally before. Now it should work. :)

**Comment by Nyholm** [comment]
Thank you

**Pull request merged**